R/plotting/access_change_area.R

```diff
@@ -0,0 +1,61 @@
+library("tidyverse")
+library("config")
+library("here")
+
+# Read files ----
+
+file_path <-
+  here("results", config::get("projected_scenario"), "agents.rds")
+agents <- read_rds(file_path)
+
+# Group agents tables ----
+
+res_vars <- c("nr_tours0", "nr_tours1",
+              "total_access0", "total_access1")
+
+agents <- agents %>%
+  group_by(area) %>%
+  summarise_at(res_vars, sum, na.rm = TRUE)
```

Comment:
`summarise_at` has been superseded by `across`, please update. :)
This comment goes for all plotting functions which use `summarise_at` or some other `_at`, `_all`, or `_if` verbs.